test-requirements.txt

```diff
@@ -1,4 +1,4 @@
 . #current directory itself, i.e. pycobertura
 coverage>=6.0,<6.4
+pytest==7.*
 mock
-pytest-cov==3.*
```

Comment:
Oh wow, so is pytest cov no longer necessary with later versions of pytest? Thanks for figuring that out!!

Reply:
No, it has to actually to do more with this: https://github.com/nedbat/coveragepy/issues/1355#issuecomment-1094407310:
```
I believe pytest-cov sets the source setting, which is why you are getting that warning about include being ignored. Do you need to use the pytest plugin?
```

Reply:
I pinned the `pytest` version to major version 7 because I thought it might make sense to version `pytest` as breaking changes in the testing could lead to sudden breakages in the test and like this we know which major version we are using.